### PR TITLE
 language selection screen -Static UI 

### PR DESCRIPTION
**Pull request by naveen-egov**

### CODE REVIEW

File: packages/digit_components/test/widgets/molecules/digit_language_card_test.dart [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      "isSelected": false
+    },
+    {
+      "label": "ENGLISH",
+      "value": "en_IN",
+      "isSelected": true
+    }
+  ]
+  """;
+void main() {
```
[the remark this answers is not in the document]
Comment (naveen-egov) [reply packages/digit_components/test/widgets/molecules/digit_language_card_test.dart:22]: @ajilo-eGov  Added missing test cases

File: packages/digit_components/lib/widgets/molecules/digit_language_card.dart [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+              ),
+              child: DigitRowCard(
+                onPressed: (data) => onPressedicon(data),
+                list: list,
+                width: (MediaQuery.of(context).size.width / list.length) -
+                    16 * list.length,
+              ),
+            ),
+            DigitElevatedButton(
+              onPressed: () => onPressedButton(),
```
[the remark this answers is not in the document]
Comment (naveen-egov) [reply packages/digit_components/lib/widgets/molecules/digit_language_card.dart]: Done

File: packages/digit_components/lib/widgets/molecules/digit_language_card.dart [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  Widget build(BuildContext context) {
+    return DigitCard(
+      margin: const EdgeInsets.only(bottom: 102, left: 8, right: 8),
+      child: ConstrainedBox(
+        constraints: const BoxConstraints.tightFor(height: 108),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.end,
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: [
+            Padding(
```
[the remark this answers is not in the document]
Comment (naveen-egov) [reply packages/digit_components/lib/widgets/molecules/digit_language_card.dart]: Done

File: apps/health_campaign_field_worker_app/lib/pages/home.dart [hunk 4]
```diff
@@ -14,7 +16,12 @@ class HomePage extends StatelessWidget {
           header: Column(
             children: const [
               BackNavigationHelpHeaderWidget(),
-              Card(child: Placeholder(fallbackHeight: 120)),
+              ProgressIndicatorContainer(
+                label: 'Just 125 more to go',
```
Comment (egov-joy) [inline apps/health_campaign_field_worker_app/lib/pages/home.dart:20]: localize these strings

**Review by egov-joy — APPROVED** [review]

**Pull request merged**